framework/src/media/audio/audio_manager.c

```diff
@@ -1109,7 +1109,11 @@ audio_manager_result_t stop_audio_stream_out(void)
 		}
 	} else {
 		if ((ret = pcm_drain(card->pcm)) < 0) {
-			meddbg("pcm_drain faled, ret = %d\n", ret);
+			if (ret == -EPIPE) {
```

Comment:
This is temp solution, we have to fix pcm_drain, itself.
```
		while (pcm->buf_idx > 0) {
			/* Wait for deque message from kernel */
			size = mq_receive(pcm->mq, (FAR char *)&msg, sizeof(msg), &prio);
			if (size != sizeof(msg)) {
				/* Interrupted by a signal? What to do? */
				return oops(pcm, EINTR, "Interrupted while waiting for deque message from kernel\n");
			}
			if (msg.msgId == AUDIO_MSG_DEQUEUE) {
				pcm->buf_idx--;
			} else if (msg.msgId == AUDIO_MSG_XRUN) {
				/* Underrun means there's no remained data */
				break;
			}
		}
```

Reply:
Then I'll close this PR